marlowe-playground-client/static/css/panels.scss

```diff
@@ -208,6 +214,55 @@ button.minus-btn:hover {
   margin-bottom: 0.5rem;
 }
 
+.action-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.action-group-explanation {
+  margin-left: 2rem;
+  line-height: normal;
+  min-width: 0;
+  flex-shrink: 1;
+  word-wrap: break-word;
+  align-self: center;
+}
+
+.action-explanation {
+  margin-left: 1rem;
+  margin-right: 1rem;
+  line-height: normal;
+  word-wrap: break-word;
+  min-width: 0;
+  flex-shrink: 1;
+}
+
+.action-label {
+  min-width: 0;
+  flex-grow: 1;
+}
+
+.action {
+  padding-left: 1rem;
+  align-self: center;
+  min-width: 0;
+  flex-grow: 1;
+  margin: 0.5rem 0;
+}
+
+.slot-input {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: normal;
+}
+
+.action-input {
+  flex-grow: 1;
+  margin-left: 0.5rem;
+  min-width: 4rem;
```

Comment:
why did you need to set this min-width?

Reply:
So that the explanation wraps around if the input is too small

Reply:
Ah, I guess I could also have set flex-shrink to 0 and a width instead, but well